packages/dx-react-scheduler-material-ui/src/templates/appointment-tooltip/layout.jsx

```diff
@@ -1,20 +1,26 @@
 import * as React from 'react';
+import { styled } from '@mui/material/styles';
 import * as PropTypes from 'prop-types';
 import Popover from '@mui/material/Popover';
-import withStyles from '@mui/styles/withStyles';
 import { SMALL_LAYOUT_MEDIA_QUERY } from '../constants';
 
-const verticalTopHorizontalCenterOptions = { vertical: 'top', horizontal: 'center' };
+const PREFIX = 'Layout';
+
+export const classes = {
+  popover: `${PREFIX}-popover`,
+};
 
-const styles = {
-  popover: {
+const StyledPopover = styled(Popover)({
+  [`& .${classes.popover}`]: {
```

Comment:
Should be without spacing. Popover class pased to the popover props

Reply:
It breaks the popover. It becomes small.